docker/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3"
+name: checkmate_stack
+services:
+  checkmate_pg:
+    env_file: ../.env
+    container_name: checkmate_pg
+    image: postgres
+    environment:
+      PGDATA: "/data/postgres"
+    volumes:
+      - ./postgres:/data/postgres
+    ports:
+      - "5432:5432"
+    restart: unless-stopped
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -d $${POSTGRES_DB} -U $${POSTGRES_USER}"]
+      interval: 1s
+      timeout: 5s
+      retries: 10
+      start_period: 20s
+  # TODO: Make it work as one stack (fix database access)
```

Comment:
As mentioned off gh, this now builds, but webapi is unable to connect to postgres and panics. I suspect this is a configuration issue, since postgres is not running on localhost from the checkmate container perspective.